main.go

```diff
@@ -19,6 +13,13 @@ import (
 	"strings"
 	"syscall"
 	"time"
+
+	"github.com/Bisnode/kubectl-login/handler"
+	"github.com/Bisnode/kubectl-login/util"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/skratchdot/open-golang/open"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
 )
 
 const version = "1.0.0"
```

Comment:
Shouldn't the version be bumped?

Reply:
Should have been bumped with the [1.0.1 release](https://github.com/Bisnode/kubectl-login/releases/tag/v1.0.1) already, but I forgot. Rather do it in a separate "release commit" than in this PR though.